lib/country_regions.rb

```diff
@@ -45,11 +45,11 @@
   vg: 'Virgin Islands, British',
   vi: 'Virgin Islands, U.S.'
 }.freeze
-LATAM_ES_COUNTRY_CODES = LATAM_ES_COUNTRY_CODE_TO_NAME.keys.map(&:to_s).freeze
+LATAM_ES_COUNTRY_CODES = LATAM_ES_COUNTRY_CODE_TO_NAME.keys.map(&:to_s).map(&:upcase).freeze
```

Comment:
It's safe to make this change because this file and its constants are currently not used anywhere in our code base.